catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogCapability.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.hadoop;
+
+import com.datastrato.gravitino.connector.capability.Capability;
+import com.datastrato.gravitino.connector.capability.CapabilityResult;
+import java.util.Objects;
+
+public class HadoopCatalogCapability implements Capability {
+  @Override
+  public CapabilityResult managedStorage(Scope scope) {
+    if (Objects.requireNonNull(scope) == Scope.SCHEMA) {
+      return CapabilityResult.SUPPORTED;
+    }
+    return CapabilityResult.unsupported("Hadoop catalog does not support managed storage.");
```

Comment:
Can you please change to "Hadoop catalog does not support managed storage for {scope}"?

Reply:
updated